scripts/enum/msg.lua

```diff
@@ -405,3 +406,4 @@ xi.msg.system =
     TRUST_NO_ENMITY              = 300, -- You cannot use Trust magic while having gained enmity.
     TRUST_SOLO_OR_LEADER         = 301, -- You cannot use Trust magic unless you are solo or the party leader.
 }
+return xi.msg
```

Comment:
I dont see returns in enum files? Is this return needed?

Reply:
accidental

Reply:
during testing log would whine about the cache, this was just to shut it the F up, and then I forgot it